### PR TITLE
Added Toggle.js, Map.js and Footer.js. Change ContactUs style to fit …

### PR DESCRIPTION
**Pull request by raeioul**
…the new funcionalities added.

### CODE REVIEW

File: src/pages/containers/Toggle.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    this.setState({
+      isHidden: !this.state.isHidden,
+    });
+  };
+  render() {
+    return (
+      <div style={styles} className="toggle-wrapper">
+        {this.state.isHidden && (
+          <button onClick={this.toggleHidden} className="button">
+            LET'S CONNECT
```
Comment (diegolanda) [inline src/pages/containers/Toggle.js]: since this translate to HTML I think we need to encode or escape it

File: src/pages/containers/Footer.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  render() {
+    return [
+      <div className="wui-footer-container">
+        <div className="wui-footer">
+          <div className="wui-footer-left">
+            <h2>About the company</h2>
+            <hr />
+            <p>
+              Nearshore Code is a software development company dedicated to build advanced solutions that meet their
+              customer's needs and requirements.
```
Comment (diegolanda) [inline src/pages/containers/Footer.js]: Same here, I think we need to escape the character

File: src/pages/containers/Toggle.js (hunk 1 above)
Comment (raeioul) [reply src/pages/containers/Toggle.js]: Done.

File: src/pages/containers/Footer.js (hunk 2 above)
Comment (raeioul) [reply src/pages/containers/Footer.js]: Fixed.

**Pull request merged**